### DISCUSSION TITLE
Docker Installation

### DISCUSSION THREAD

**Discussion by satoricode** [nilsteampassnet/TeamPass#3870] [General]
Are there a more detailed installation guide for Docker (for inexperienced docker users).  Would like to TeamPass running on Docker (defaults) and struggling with the docker-compose.yml

Any help appreciate - also is TeamPass actively been added to and stable as seems like an awesome solution

**Comment by cos-gl** [comment]
Hi @satoricode,
I'm not a docker expert, but this is what i've done on my Ubuntu 22.04 machine

#### Install docker
```
apt update
apt install docker.io docker-compose
```

#### Download docker compose 
```
wget https://github.com/nilsteampassnet/TeamPass/blob/master/docker-compose.yml
```

#### Modify docker-compose.yml file to suit your env (look at this values) 
```
VIRTUAL_HOST: teampass.yourdomain.local
CERT_NAME: teampass.yourdomain.local.crt
MYSQL_PASSWORD: YOUR_SECRET_PASSWORD
```
About the certificate read the instructions on the main page: https://github.com/nilsteampassnet/TeamPass#with-docker-compose

#### Configure docker and start containers 
```
docker network create backend
docker-compose up -d
docker container ls 
```

Check your browser at https://teampass.yourdomain.local and follow Note1 e Note2 in the instructions https://github.com/nilsteampassnet/TeamPass#with-docker-compose

**Reply by nilsteampassnet** [reply]
@cos-gl Thank you.
If you don't mind I will add your inputs in the documentation.

**Reply by cos-gl** [reply]
> @cos-gl Thank you. If you don't mind I will add your inputs in the documentation.

@nilsteampassnet for sure. Feel free to add what you want in the documentation.

**Comment by Hamfankam** [comment]
The container has not been updated for 7 months, please update it to the latest current version! Thank you for your project! https://hub.docker.com/r/dormancygrace/teampass/tags
